test/src/device-request-prompt.spec.ts

```diff
@@ -60,4 +59,46 @@ describe('device request prompt', function () {
       })
     ).rejects.toThrow(TimeoutError);
   });
+
+  it('can be opened', async function () {
+    const {page, httpsServer} = state;
+
+    await page.goto(httpsServer.PREFIX + '/device-request.html');
+
+    await expect(
+      Promise.all([
+        page.waitForDevicePrompt({timeout: 100}),
+        page.click('#bluetooth'),
+      ])
+    ).resolves.toBeTruthy();
+  });
+
+  it('can be cancelled', async function () {
```

Comment:
Let's wrap all the `it`s till here in a `describe('waitForDevicePrompt')`.

Reply:
done in https://github.com/puppeteer/puppeteer/pull/11185/commits/2f98246c09274b82e714b14dc77c87caefbfe7e9